proc/proc_windows.go

```diff
@@ -26,7 +26,7 @@ type OSProcessDetails struct {
 }
 
 // Launch creates and begins debugging a new process.
-func Launch(cmd []string) (*Process, error) {
+func Launch(cmd []string, wd string) (*Process, error) {
```

Comment:
It looks like we are rewriting os/exec.Cmd. Can we just use os/exec.Cmd here instead?